.azure-pipelines/packages.yml

```diff
@@ -97,7 +97,7 @@ stages:
         arguments: --configuration $(buildConfiguration)
 
     - task: DotNetCoreCLI@2
-      displayName: dotnet publish Datadog.Trace.ClrProfiler.Managed
+      displayName: dotnet publish Datadog.Trace.ClrProfiler.Managed 2.0
```

Comment:
nit: I would prefer writing out `netstandard2.0` and `netcoreapp3.1` here and in the other tasks in this build pipeline to make them names as descriptive as possible

Reply:
Fixed